appcues/src/main/java/com/appcues/Storage.kt

```diff
@@ -13,6 +13,7 @@ internal class Storage(
 
     private enum class Constants(val rawVal: String) {
         DeviceId("appcues.deviceId"),
+        DeviceToken("appcues.deviceToken"),
```

Comment:
I was referring to the existing reference from the  Hackaton but I was wondering if it would be clearer to name this PushToken

Reply:
yes, lets please name this `PushToken` and use `pushToken` on L33. Landed on that consensus in chat I believe, and matches iOS side https://github.com/appcues/appcues-ios-sdk/blob/sdk4/Sources/AppcuesKit/Data/Storage.swift#L31